### PR TITLE
about page updates

### PR DESCRIPTION
**Pull request by gumaerc**
### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/4641

### Description (What does it do?)
This PR adds some updates the copy and styles on the About Us page. The initial version of the About Us page had the wrong typography rules based on a miscommunication in Figma. The sub headings have been updated to `h4` style, and the body text is now `body1`. Previously in Figma the headings were `h5` and the body text was `body3`. I also updated these styles on the terms of service and privacy policy pages.

Document fragment identifiers were also added to the headers of the various About Us page sections, and a link was added to the home page search box on the "Non-Degree Education" text that links to the appropriate section of the About Us page.

### Screenshots (if appropriate):
image
image

### How can this be tested?
 - Spin up `mit-open` on this branch
 - Visit the home page at http://localhost:8063/
 - Click the "Non-Degree Education" link
 - Verify that you are brought to http://localhost:8063/about#non-degree-learning
 - Verify that the design of the About Us page looks as it should according to the design linked in the issue about at both desktop and mobile resolutions

### CODE REVIEW

File: frontends/mit-open/src/pages/AboutPage/AboutPage.tsx [hunk 1]
```diff
@@ -91,50 +121,133 @@ const AboutPage: React.FC = () => {
...
-              they come from.
-            </SubHeaderText>
+            <SubHeaderTextContainer>
+              <Typography variant="subtitle1">
+                Our non-degree learning resources empower you to:
+              </Typography>
+              <ul>
+                <li>
+                  Access content from world-renowned faculty and experts much of
+                  it for free
```
Comment (pdpinch) [inline frontends/mit-open/src/pages/AboutPage/AboutPage.tsx:148]:
```suggestion
                  Access content from world-renowned faculty and experts – much of
                  it for free
```

File: frontends/mit-open/src/pages/AboutPage/AboutPage.tsx [hunk 2]
```diff
@@ -91,50 +121,133 @@ const AboutPage: React.FC = () => {
...
+                <li>
+                  Build job-relevant skills through application-focused
+                  offerings{" "}
+                </li>
+                <li>
+                  Experience peer-to-peer interactions and grow your network
+                </li>
+                <li>
+                  Receive a certificate from MIT, or just learn for your own
+                  sake
```
Comment (pdpinch) [inline frontends/mit-open/src/pages/AboutPage/AboutPage.tsx:159]:
```suggestion
                  Receive a certificate from MIT, or just learn for your own
                  enjoyment
```

File: frontends/mit-open/src/pages/AboutPage/AboutPage.tsx [hunk 3]
```diff
@@ -91,50 +121,133 @@ const AboutPage: React.FC = () => {
...
+                </li>
+                <li>
+                  Learn from the same materials used by MIT students on campus
+                </li>
+                <li>
+                  Download, reuse and edit thousands of learning resources
```
Comment (pdpinch) [inline frontends/mit-open/src/pages/AboutPage/AboutPage.tsx:165]:
```suggestion
                  Download, share and modify thousands of learning resources
```

File: frontends/mit-open/src/pages/AboutPage/AboutPage.tsx [hunk 4]
```diff
@@ -91,50 +121,133 @@ const AboutPage: React.FC = () => {
...
+              What is non-degree learning at MIT?
+            </BodyText>
+            <BodyText variant="body1">
+              MIT's non-degree learning programs provide specific skills,
+              knowledge, or certifications without the time commitment and
+              breadth of study required for a full degree. These programs are
+              designed to be flexible and accessible, allowing professionals,
+              students, and lifelong learners to engage with MIT's educational
+              offerings from anywhere in the world. Whether you're looking to
+              upskill, explore a new field, or gain a deeper understanding of a
```
Comment (pdpinch) [inline frontends/mit-open/src/pages/AboutPage/AboutPage.tsx:206]:
```suggestion
              offerings from anywhere in the world. Many of these resources are free or low-cost.  Whether you're looking to
              upskill, explore a new field, or gain a deeper understanding of a
```

File: frontends/mit-open/src/pages/AboutPage/AboutPage.tsx [hunk 5]
```diff
@@ -91,50 +121,133 @@ const AboutPage: React.FC = () => {
...
+              subject, MIT's non-degree learning resources provide a pathway for
+              personal and professional growth.
+            </BodyText>
+          </BodySection>
+          <BodySection>
+            <BodyText
+              variant="h4"
+              id={WHAT_KINDS_OF_CONTENT_FRAGMENT_IDENTIFIER}
+            >
+              What kinds of content are available from MIT Open?
```
Comment (pdpinch) [inline frontends/mit-open/src/pages/AboutPage/AboutPage.tsx:216]:
```suggestion
              What kind of content is available from MIT Open?
```

File: frontends/mit-open/src/pages/AboutPage/AboutPage.tsx [hunk 6]
```diff
@@ -91,50 +121,133 @@ const AboutPage: React.FC = () => {
...
+            </BodyText>
+            <BodyText variant="body1">
+              MIT is dedicated to advancing knowledge beyond students enrolled
+              in MIT's campus programs. Several divisions within MIT offer
+              educational opportunities accessible to learners worldwide,
+              catering to a diverse range of needs.
```
Comment (pdpinch) [inline frontends/mit-open/src/pages/AboutPage/AboutPage.tsx:222]:
```suggestion
              MIT Open organizes non-degree educational opportunities into two 
              categories: academic and professional.
```

File: frontends/mit-open/src/pages/AboutPage/AboutPage.tsx [hunk 7]
```diff
@@ -91,50 +121,133 @@ const AboutPage: React.FC = () => {
...
+              <br />
+              <br />
+              MIT's ACADEMIC courses, programs, and materials mirror MIT
```
Comment (pdpinch) [inline frontends/mit-open/src/pages/AboutPage/AboutPage.tsx:225]:
```suggestion
              MIT's Academic courses, programs, and materials mirror MIT
```

File: frontends/mit-open/src/pages/AboutPage/AboutPage.tsx [hunk 8]
```diff
@@ -91,50 +121,133 @@ const AboutPage: React.FC = () => {
...
+              curriculum and residential programs, making these available to a
+              global audience. Approved by faculty committees, academic content
+              furnishes a comprehensive foundation of knowledge, skills, and
+              abilities for students pursuing their academic objectives.
+              Renowned for its rigor and challenge, MIT's academic offerings
+              deliver an experience on par with the campus environment.
+              <br />
+              <br />
+              MIT's PROFESSIONAL courses and programs are tailored for working
```
Comment (pdpinch) [inline frontends/mit-open/src/pages/AboutPage/AboutPage.tsx:234]:
```suggestion
              MIT's Professional courses and programs are tailored for working
```

**Review by pdpinch — COMMENTED** [review]
Some last-minute copy edits. It would be great if they can be included. Not a big deal if they have to wait.

**Comment by gumaerc** [comment]
@pdpinch All set.

**Review by abeglova — APPROVED** [review]
👍

**Pull request merged**